scripts/functions-deploy-tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Function Deploy Integration Test
+
+Function deploy integration test cylcles through "create -> update -> update -> ..." phases to make sure all supported function triggers are deployed with correct configuration values.
+
+The test isn't "thread-safe" - there should be at most one test running on a project at any given time. I suggest you to use your own project to run the test.
+
+You can set the test project and run the integration test as follows:
+
+```bash
+$  GCLOUD_PROJECT=${PROJECT_ID} npm run test:functions-deploy
+```
+
+The integration test blows away all existing functions! Don't run it on a project where you have functions you'd like to keep.
```

Comment:
Uh... This is kinda a big deal right here.

Can you like, make this in bold, and red... and marquee?